CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## [Unreleased]
 
+## [3.3.0] - 2023-08-27
+
+### General
+
+- Added documentation for PaginatedList
+- Rework requester URLs to accomodate graphql and new quizzes endpoints (Thanks, [@bennettscience](https://github.com/bennettscience))
+
+### Bugfixes
+
+- Fixed PaginatedList not respecting new quizzes endpoints (Thanks, [@matthewf-ucsd](https://github.com/matthewf-ucsd))
```

Comment:
Thanks for the mention but wrong Matthew! 😁

Reply:
Ack, there are too many of us! This is what I get for pushing code at 2am 🤦.

Sorry about that, will fix in another push to develop.